pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -50,10 +57,13 @@ type capability struct {
 }
 
 var _ capabilityIface = (*capability)(nil)
+var _ capabilities.ConsensusCapability = (*capability)(nil)
+var OCR3CapabilityValidator = capabilities.NewValidator[CapabilityConfig, CapabilityInputs, CapabilityOutputs](capabilities.ValidatorArgs{Info: info})
```

Comment:
Hmmmm... does this need to be public, and do we need to instantiate at the package level (i.e. can we just move it into newCapability)?

Reply:
Another reason for public exposure + package level is to have an easy way to generate all of the test fixtures by importing all of the validators in a single file, rather than have a generate script in each package. WDYT?

Reply:
Yeah if you want to test things in isolation I think it's fine to leave it at the package level -- still not sure it needs to be public though? In the end we embed the validator into the capability, and I think it's nice to ensure that we only access it via there.

Reply:
The value I see exposing it publically would be to generate all test fixtures without having to create an instance of the capability first.

Maybe this isn't a big deal though, I was just thinking it was extra steps if some capabilities are only creatable via a `NewCapability` type function that didn't allow you to pass in empty args.

Reply:
Hmmm... fair point -- you can ignore this then 👍